lib/sidekiq/api.rb

```diff
@@ -491,8 +490,8 @@ def serialized_global_id?(hash)
     end
 
     def uncompress_backtrace(backtrace)
-      decoded = Base64.decode64(backtrace)
-      uncompressed = Zlib::Inflate.inflate(decoded)
+      base64_decoded = backtrace.unpack1("m")
```

Comment:
I think you should use `m0` for pack/unpack here. We don't want or need newlines.

Reply:
Sure, changed. Tests are still passing